tutorpod_autoscaling/patches/k8s-override

```diff
@@ -1,92 +1,25 @@
-{%- if POD_AUTOSCALING_LMS_HPA %}
+{% for service, autoscaling in iter_autoscaling_config() %}
+{%- if autoscaling["enable_hpa"] %}
```

Comment:
I think I would turn this `for/if` to a single `for service, autoscaling in iter_hpa_config` loop, and a similar function for VPA, but of course that's up to you.